src/it/resources/org/checkstyle/suppressionxpathfilter/classtypeparametername/InputXpathClassTypeParameterNameUpperCase.java

```diff
@@ -0,0 +1,7 @@
+package org.checkstyle.suppressionxpathfilter.classtypeparametername;
+
+public class InputXpathClassTypeParameterNameUpperCase {
+    class MyClass1<aBc> {} // warn
+    class MyClass2<LISTENER> {}
```

Comment:
Unfortunately all examples are the same by structure.
In xpath value is in different structure of xpath expressions.
Please make second example to be classes in interface.
Third example is classes in methods.